services/workflows-service/jest.config.cjs

```diff
@@ -1,14 +1,26 @@
 module.exports = {
-  preset: 'ts-jest',
   testEnvironment: 'node',
-  modulePathIgnorePatterns: ['<rootDir>/dist/'],
   testRegex: '(/__tests__/.*|(\\.|/)(unit|e2e|intg)\\.test)\\.ts$',
+
+  preset: 'ts-jest',
+  modulePathIgnorePatterns: ['<rootDir>/dist/'],
+  modulePaths: ['<rootDir>/src'],
+  moduleDirectories: ['node_modules'],
   moduleNameMapper: {
     '^@/(.*)$': '<rootDir>/src/$1',
   },
+
+  transform: {
+    '^.+\\.{ts}?$': [
+      'ts-jest',
+      {
+        tsconfig: 'tsconfig.test.json',
+      },
+    ],
+  },
```

Comment:
boy scout: 
Resolved Jest warning by updating deprecated configuration key. 

this prevents the warning: '[jest-config].globals.ts-jest.tsConfig' is deprecated, use '[jest-config].globals.ts-jest.tsconfig' instead.